index.html

```diff
@@ -0,0 +1,117 @@
+      </div>
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <a class="link" href="index.html">Home</a>
+        <a class="link" href="about.html">About</a>
+        <a class="link" href="#">Programs</a>
+        <a class="link" href="#">Join</a>
+        <a class="link" href="#">Sponsors</a>
+        <a class="link" href="#">News</a>
+        <a href="#" id="campaign" class="link">YCI LAUNCH</a>
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="landing-page-section">
+      <p class="greetings">"Hello! Welcome to YCI"</p>
+      <h1 class="landing-page-header">YOUNG CODERS INITIATIVE<br>2022</h1>
+      <p class="landing-page-para">
+        Young Coders Initiative (YCI) was established to help seconandary school students learn 
+        how to code and create Software programs from a tender age. Young Coders Initiative 
+        aims at equipping secondary school students with skills and knowledge in the field of computer science. 
+        YCI national launch will take place in December, 2022, in Malawi.
+      </p>
+      <h2 class="duration">2022.12.12 (SAT) ~ 13(SUN)</h2>
+      <p class="location">@ Comesa Hall, Blantyre, Malawi</p>
+    </section>
+
+    <section class="activities">
+      <h2 class="activities-header">Main Program</h2>
+      <div class="row">
+        <div class="col-row">
+          <img class="presentation" src="./img/lecture.png" alt="Lecture">
+          <h3 class="lecture-heading">Founder's Speech</h3>
+          <p class="lecture-para">
+            The founder will describe the motive behind the initiative and
+            he will also share his vision and mission for YCI with the audience.
+        .</p>
+        </div>
+        <div class="col-row">
+          <img class="forum" src="./img/forum.png" alt="Forum">
+          <h3 class="forum-heading">Pioneer's Speech</h3>
+          <p class="forum-para">
+            The first students of YCI will share their thoughts about this initiative.
+            They will describe how the program has benefited them so far.
+          </p>
+        </div>
+        <div class="col-row">
+          <img class="connect" src="./img/connect.png" alt="Connect">
+          <h3 class="ignite-heading">YCI Programs showcase</h3>
+          <p class="ignite-para">
+            This time will be for programs presentation. This time will be used to present 
+            the list of programs and services that will be offered by this initiative. 
+          </p>
+        </div>
+      </div>
+      <div class="join-summit">
+        <button class="join" type="button">Join YCI</button>
+        <a href="#" class="see-program">SEE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <section class="speakers-container">
+      <h2 class="speakers-header">SPEEKERS</h2>
+      <div class="speakers-grid-container">
+
+      </div>
+    </section>
+
+  </main>
+
+  <footer>
```

Comment:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using section and footer tags 💪 We must not forget to use the article tag! Kindly make the necessary changes in your speaker's HTML dynamic creation so that each of them has an article tag 🤓